app/src/main/java/org/kiwix/kiwixmobile/main/KiwixTextToSpeech.java

```diff
@@ -182,8 +182,10 @@ public void pauseOrResume() {
     if (currentTTSTask.paused) {
       if (!requestAudioFocus()) return;
       currentTTSTask.start();
+      MainActivity.sendNotification("pause",context);
```

Comment:
There should be a space after the comma. To avoid these type of mistakes import the Square's code style and reformat your code using `Ctrl + Alt + L` before committing.

Reply:
Doing this did a lot of changes: 2028 additions and 2035 deletions. Is it fine?